async-query-core/src/main/antlr/SqlBaseLexer.g4

```diff
@@ -134,6 +134,7 @@ AS: 'AS';
 ASC: 'ASC';
 AT: 'AT';
 AUTHORIZATION: 'AUTHORIZATION';
+BEGIN: 'BEGIN';
```

Comment:
The antlr changes already merged in https://github.com/opensearch-project/sql/pull/2787, could you rebase?

Reply:
sure